### PR TITLE
Added an explicit information that find/findOne selector is a CSS selector

### PR DESCRIPTION
**Pull request by mlewand**
## What is the purpose of this pull request?

Task

## Does your PR contain necessary tests?

All patches which change the editor code must include tests. You can always read more
on [PR testing](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_contributing_code-section-tests),
[how to set the testing environment](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_tests) and
[how to create tests](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_tests-section-creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [ ] Manual tests

Skipped tests, this is doc only change.

## What changes did you make?

Added an explicit information that selector is a CSS selector.

During the process I was thinking about higher DRY factor, however both methods have such documentation that it differs in details here and there, thus rendering `@inheritdoc` unfeasible.

Closes ckeditor/ckeditor-docs#43.

### CODE REVIEW

File: core/dom/element.js [hunk 1]
```diff
@@ -2044,20 +2044,20 @@ CKEDITOR.dom.element.clearMarkers = function( database, element, removeFromDatab
 		},
 
 		/**
-		 * Returns first element within this element that matches specified `selector`.
+		 * Returns the first element within this element that matches specified `selector`.
```
Comment (AnnaTomanek) [inline core/dom/element.js]: Returns the first element within this element that matches the specified `selector`.

File: core/dom/element.js [hunk 2]
```diff
@@ -2044,20 +2044,20 @@ CKEDITOR.dom.element.clearMarkers = function( database, element, removeFromDatab
...
 		 *
 		 * **Notes:**
 		 *
 		 *	* Not available in IE7.
-		 *	* Unlike native `querySelectorAll` this method ensures selector contextualization. This is:
+		 *	* Unlike native `querySelector` this method ensures selector contextualization. This is:
```
Comment (AnnaTomanek) [inline core/dom/element.js]: Unlike the native `querySelector`, (...)

**Comment by mlewand** [comment]
Despite it being accidentally merged @msamsel please still have a look at it making sure that all is good.

**Comment by msamsel** [comment]
@mlewand, @AnnaTomanek it looks fine from my perspective :) 👍

**Pull request merged**